Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift

```diff
@@ -61,6 +61,11 @@ import Foundation
             "card_holder_phone_number": cardData.cardHolderPhoneNumber ?? ""
```

Comment:
sorry I missed this before, as it's not part of the changes. should this be removed in the first place?

Reply:
@azaky please check a new PR, we deleted it